server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/Paragraph.java

```diff
@@ -24,7 +24,7 @@ public class Paragraph extends CFASTNode {
   String snippet;
   String name;
 
-  public Paragraph(String snippet, String name, Location location) {
+  public Paragraph(String name, String snippet, Location location) {
```

Comment:
not sure what the point of this change was...

Reply:
So it aligns with the Section constructor's argument order. To reduce cognitive load :)